client/components/AllProduct.js

```diff
@@ -1,24 +1,24 @@
 import React from 'react'
 import {connect} from 'react-redux'
 import {getAllProductsThunk} from '../store/allProducts'
+import {Link} from 'react-router-dom'
 
 class AllProducts extends React.Component {
   componentDidMount() {
     this.props.loadDucks()
   }
 
   render() {
-    console.log('PROPS: ', this.props)
     return (
       <div className="allProducts">
         <h1>All Producks</h1>
         <ul>
```

Comment:
Maybe we don't use the <ul> anymore? We could just wrap it in a div instead.

Reply:
sorry I think I'm confused with how the layout works for the comments on here. Which part do you think we could wrap in a div?

Reply:
God sorry these comments were terrible. I wrote something in code-like language, and it must have inserted it or something annoying like that. We meant "ul". We maybe don't need an unordered list there, just a div.

Reply:
yeah i agree, I started playing around with css and a div would be better for cleaner display